integration-templates/checkr-partner/mappers/to-background-check.ts

```diff
@@ -0,0 +1,13 @@
+import type { CheckrTriggeredBackgroundCheck, BackgroundCheck } from '../../models';
```

Comment:
english: not sure about the term mappers but it's not my main language, I would call them formatters maybe

Reply:
Thanks, IMO `mappers` is clear